cypress/integration/v2.spec.ts

```diff
@@ -0,0 +1,25 @@
+import {v4 as UUIDv4} from 'uuid'
+
+context('V2 Testing', () => {
+  before(() => {
+    cy.setupMetaMask()
+  })
+
+  after(() => {
+    cy.lockMetaMask()
+  })
+
+  it('foo', () => {
+    cy.visit('/#/v2/create')
+    const projectName = UUIDv4()
+    cy.findByLabelText(/project name/i).type(projectName)
+
+    cy.findByRole('button', {  name: /next: funding cycle/i}).click()
+    cy.findByRole('button', {  name: /next: review and deploy/i}).click()
+    cy.findByRole('button', {  name: /connect wallet to deploy/i}).click()
+    cy.findByRole('button', {  name: /metamask metamask/i}).click()
+    cy.acceptMetaMaskAccess()
+    cy.findByRole('button', {  name: /deploy project to .*/i}).click()
+    cy.confirmMetaMaskTransaction()
```

Comment:
**question** wondering what your thoughts are on somehow asserting the final result on-chain. The fact the transaction was successful is definitely meaningful, but we'll also need to eventually test that "when you set the slider to 100%, it submits a payload with data `10000`" for example.

Can def be a follow-up. Not sure if we should test that at the cypress/e2e level or whether its better for jest/integration level.

Reply:
Excuse my noobiness, but is there a way we can query the blockchain for the transaction? We could just look at doing that too. Might be best left to int/unit though - worth discussing

Reply:
yeah i reckon maybe not E2E worthy.

Etherscan API is one possible approach to fetching real tx data: https://etherscan.io/apis#transactions

Otherwise we could do a legit read of the Juicebox contracts to check the various parameters are set correctly.
Juicebox docs are here https://info.juicebox.money/docs/protocol/api/contracts/jbprojects/read/tokenuri